stable/aws-node-termination-handler/values.yaml

```diff
@@ -8,6 +8,9 @@ image:
   pullPolicy: IfNotPresent
   pullSecrets: []
 
+runAsUserID: 1000
```

Comment:
Recommend putting these under a `securityContext` subfield in order to group related values appropriately.